### DISCUSSION TITLE
How to connect with shield?

### DISCUSSION THREAD

**Discussion by kud0h** [Z3d0X/filament-logger#113] [Q&A] [answered]
How do I connect this plugin with Filament Shield? I have generated the policy (shield) for the activity, but all roles can still access the menu and pages.

**Comment by kigathi-chege** [comment, marked as answer]
@kud0h 

I found the solution. Put this in the `boot` method of your `AppServiceProvider`

`Gate::policy(Activity::class, ActivityPolicy::class);`

**Reply by marcosmarcolin** [reply]
Thanks @kigathi-chege, solved for me too!
